components/core/config/schemas.txt

```diff
@@ -1,19 +1,98 @@
-// Delimiters
-delimiters: \t\r\n!"#$%&'\(\)\*,:;<>?@\[\]\^_`\{\|\}~
+// ===
+// Timestamps
```

Comment:
Does this cover all the known timestamp patterns in `TimestampPattern.cpp`?

Reply:
It is best effort, but all patterns in `TimestampPattern.cpp` should be in the schema now. The timestamps requiring log boundary knowledge are categorized separately and will be addressed in the future.